frontend/src/services/AttendanceService.ts

```diff
@@ -1,9 +1,28 @@
-import { AttendanceApi, AttendanceControllerDeleteAttendanceRequest } from './api/apis/AttendanceApi';
+import {
+  AttendanceApi,
+  AttendanceControllerDeleteAttendanceRequest,
+  AttendanceControllerUpdateAttendanceTimesRequest
+} from './api/apis/AttendanceApi';
 import { AttendanceResponse } from './api/models';
 
 import ApiConfigStore from './ApiConfigStore';
 import { Configuration } from './api/runtime';
 
+export async function updateAttendanceTimes(attendeeId: number, eventId: number, startTime: string, endTime: string) {
```

Comment:
A comment about how updating startTime and endTime will also update attendee's induction points?  (which are automatically calculated from startTime and endTime)